metadata/2024-08-30-enable-v7/enable_bytecode_v7.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable bytecode version 7",
+  "description": "Enable bytecode version 7",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/blob/main/sources/2024-08-30-enable-v7",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core/pull/14124"
```

Comment:
this should be an link to an AIP issue - is there an AIP for this?

Reply:
Bytecode v7 does not have a standalone AIP so added the link to the specific PR. @wrwg may have more context here.

Reply:
v7 was introduced the first time for resource access control, but that feature is now disabled by a new separate feature flag, as are enum types. As a background, we could link to both AIP-91 (enum) and AIP-56 (resource access control). But really there should not be an AIP needed, turning on support for v7 is more similar like a feature gated refactoring.